src/gui/qgsoptionsdialogbase.cpp

```diff
@@ -396,10 +397,15 @@ void QgsOptionsDialogBase::addPage( const QString &title, const QString &tooltip
       mOptTreeModel->appendRow( item );
   }
 
+  QgsScrollArea *scrollArea = new QgsScrollArea();
+  scrollArea->setWidgetResizable( true );
+  scrollArea->setFrameShape( QFrame::NoFrame );
+  scrollArea->setWidget( widget );
```

Comment:
Should we first check if widget isn't a QScrollArea or is a widget with a layout which directly contains a QScrollArea first? I'm not sure what happens when you place a scrollarea directly inside a scroll area, but I'm sure it's not pretty...

Reply:
It actually works fine. No visual nor behavioral artifact for me. I'd suggest we just remove the scrollareas from existing pages for cleaner code and not sweat about the odd nested scroll area left over or added by a plugin.

Reply:
ok, fair enough!